src/handlers/comments/getComments.js

```diff
@@ -54,4 +49,15 @@ ${entries.join('\n')}
 `;
 }
 
+function parseAdditionalFieldsJSON(items) {
+    return items.map((item) => {
+        try {
+            item.additional = JSON.parse(item.additional);
+            return item;
+        } catch (e) {
+            return '{}';
```

Comment:
Ah sorry, I meant to have `item.additional` be `{}` in that case, so along those lines:

```js
try {
    item.additional = JSON.parse(item.additional);
}
catch (_) {
    item.additional = {};
}

return item;
```